CPScripts/watchdog.sh

```diff
@@ -69,52 +192,26 @@ fi
 
 while [ true = true ]
 	do
-		systemctl status $NAME 2>&1>/dev/null
```

Comment:
I moved all of this into its own function so that a service can have conditions for being checked.